### ISSUE TITLE
Activiteiten overzicht

### ISSUE THREAD

**Issue by jeroenvandiepen** [svIndicium/indicium-dashboard#99]
Activiteiten overzicht geeft geen activiteiten weer

ICS werkt wel gewoon

**Comment by joostlek** [comment]
Yes, de eventmanager applicatie is momenteel nog niet gedeployed, en voor de normale website wordt nog gebruik gemaakt van een ouder CMS. Dus als de eventmanager afgemaakt is, wordt die gedeployed en werkt het weer
